### PR TITLE
behdad-fonts: init at v0.0.3

### PR DESCRIPTION
**Pull request by LinArcX**
###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nix-review --run "nix-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Assured whether relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/data/fonts/behdad-fonts/default.nix [hunk 1]
```diff
@@ -0,0 +1,26 @@
+{ stdenv, fetchFromGitHub }:
+
+stdenv.mkDerivation rec {
+  name = "behdad-fonts";
```
Comment (romildo) [inline pkgs/data/fonts/behdad-fonts/default.nix]: Use `pname` instead of `name`.

File: pkgs/data/fonts/behdad-fonts/default.nix [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+  };
+
+  installPhase = ''
+    mkdir -p $out/share/fonts/behdad-fonts
+    cp -v $( find . -name '*.ttf') $out/share/fonts/behdad-fonts
+  '';
+
+  meta = with stdenv.lib; {
+    homepage = https://github.com/font-store/BehdadFont;
+    description = "A Persian/Arabic Open Source Font - بهداد: فونت فارسی با مجوز آزاد";
```
Comment (veprbl) [inline pkgs/data/fonts/behdad-fonts/default.nix]: I think we should stick with ASCII here.

**Comment by veprbl** [comment]
cc @volth

File: pkgs/data/fonts/behdad-fonts/default.nix (hunk 2 above)
Comment (LinArcX) [reply pkgs/data/fonts/behdad-fonts/default.nix]: Why?
Is there any technical issue with it?

Comment (volth) [reply pkgs/data/fonts/behdad-fonts/default.nix]: Some non-ASCII are already in descriptions (and maintainer names), for example
https://github.com/NixOS/nixpkgs/blob/master/pkgs/data/fonts/caladea/default.nix#L26

Right-to-left script might help to spot a bug, in Hydra web UI or something like this

**Pull request merged**